package.json

```
@@ -13,6 +13,10 @@
   "license": "CC BY-SA 4.0",
   "author": "Open Terms Archive",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint:markdown": "markdownlint *.md content/*.md content/**/*.md content/**/**/*.md "
```

This is redundant. The glob syntax should be enough for traversal:

```suggestion
    "lint:markdown": "markdownlint **/*.md"
```

---

`markdownlint **/*.md` results in linting all markdown files from the repository which is long and unnecessary (the `node_modules` folder for example!) and does not lint the files in the root path (like the `README.md`).

So I changed by `markdownlint *.md content/**/*.md` 👍